packages/@react-spectrum/searchwithin/stories/SearchWithin.stories.tsx

```diff
@@ -119,7 +119,9 @@ CustomWidth30.storyName = 'Custom width: 30';
 export const LabelPositionSide = () => render({labelPosition: 'side'});
 LabelPositionSide.storyName = 'labelPosition: side';
 
-export const NoLabel = () => render({label: undefined, 'aria-label': 'Aria Label'});
+export const NoVisibleLabel = () => render({label: undefined, 'aria-label': 'Search'});
```

Comment:
Nit: For this story and the default render story, it maybe better to have the aria-label/label be something different than "Search" just to make it easier to distinguish between the `aria-label="Search"` that gets applied to the SearchField automatically and the one provided by the user. Right now VoiceOver states "Search search" when focusing the SearchField portion of the SearchWithin component which is kinda confusing.